scripts/docker/interactive/build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+
+# shellcheck disable=SC2048
```

Comment:
I forced bash instead of sh because I'm using `BASH_SOURCE` to find the script directory but we can change it to `dirname $0` it will work in most cases. 

I like to use `shellcheck` since bash has many pitfalls 😅 I suppose it wouldn't hurt to have it in the CI.